web/settings.html

```diff
@@ -180,6 +180,19 @@ <h2>Settings</h2>
                             </div>
                         </div>
 
+                        <div class="form-group row">
+                            <label for="frmFXServer-infinity" class="col-sm-3 col-form-label">OneSync Infinity</label>
+                            <div class="col-sm-9">
+                                <label class="switch switch-label switch-pill switch-success fix-pill-form">
+                                    <input class="switch-input" type="checkbox" id="frmFXServer-infinity" {{fxserver.infinity}} {{disableWrite}}>
+                                    <span class="switch-slider" data-checked="On" data-unchecked="Off"></span>
+                                </label>
+                                <span class="form-text text-muted">
+                                    Required if you server has more than 128 slots.
```

Comment:
Change it **from**: "Required if you server has more than 128 slots."
**To**: "Required if your server has more than 128 slots and supports Onesync Infinity."

Reply:
Yeah. Done!